tests/http/env-value-with-multiple-underscores.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Env value with underscores
```

Comment:
can we merge all http_spec tests into single one? No need to make many very similar tests

Reply:
done